### PR TITLE
add ts SDK example for executing and fetching query results in one call

### PR DESCRIPTION
**Pull request by vlad1chek**
- add community ts sdk snippet of getting query execution result

before:
image

after:
image

### CODE REVIEW

File: api-reference/executions/endpoint/get-execution-result.mdx [hunk 1]
```diff
@@ -18,7 +18,9 @@ Result returns the status, metadata, and query results (in JSON) from a query ex
 <Accordion title="Execute query and get result in one call">
   <Tip> If you are using the [Python SDK](https://github.com/duneanalytics/dune-client/tree/d2195b2a9577e2dcae5d2600cb3eddce20987f38), you can directly executes and fetches result in one function call, like below: </Tip>
```
Comment (Andriy-Shymkiv) [inline api-reference/executions/endpoint/get-execution-result.mdx:19]: Would be good to add a link for the [Typescript SDK](https://github.com/bh2smith/demo-ts-dune-client) here

Comment (vlad1chek) [reply api-reference/executions/endpoint/get-execution-result.mdx:19]: fell free to fix 🙌

File: api-reference/executions/endpoint/get-execution-result.mdx [hunk 2]
```diff
@@ -16,9 +16,11 @@ Result returns the status, metadata, and query results (in JSON) from a query ex
 </Info>
 
 <Accordion title="Execute query and get result in one call">
-  <Tip> If you are using the [Python SDK](https://github.com/duneanalytics/dune-client/tree/d2195b2a9577e2dcae5d2600cb3eddce20987f38), you can directly executes and fetches result in one function call, like below: </Tip>
+  <Tip> If you are using the [Python SDK](https://github.com/duneanalytics/dune-client/tree/d2195b2a9577e2dcae5d2600cb3eddce20987f38) or community [Typescript SDK](https://github.com/bh2smith/ts-dune-client), you can directly executes and fetches result in one function call, like below: </Tip>
```
Comment (nzrii-kysel) [inline api-reference/executions/endpoint/get-execution-result.mdx:19]: could fix the obvious typos as well

Comment (vlad1chek) [reply api-reference/executions/endpoint/get-execution-result.mdx:19]: nice catch! please fix 🙂

**Review by agaperste — APPROVED** [review]
LGTM! Thanks for adding!

**Pull request merged**